docs/pages/versions/unversioned/sdk/firebase-recaptcha.md

```diff
@@ -282,7 +274,7 @@ export default function PhoneAuthScreen() {
           textContentType="telephoneNumber"
           placeholder="+1 999 999 9999"
           editable={!verificationId}
-          onChangeText={(phoneNumber: string) => setPhoneNumber(phoneNumber)}
+          onChangeText={phoneNumber => setPhoneNumber(phoneNumber)}
```

Comment:
```suggestion
          onChangeText={setPhoneNumber}
```
We can simplify that prop, since `onChangeText` always return string and only has one parameter.

Same with other `onChangeText` usages, but we can also leave it as is.